setup.py

```diff
@@ -20,7 +20,7 @@
         'pytest-runner'] if any(x in ('pytest', 'test') for x in sys.argv) else [],
     tests_require=['pytest>=6.2.4', 'shapely', 'neobolt'],
     install_requires=['neo4j-driver==4.3.6', 'pytz>=2021.1',
-                      "neobolt==1.7.17", "Shapely==1.7.1", "six==1.16.0"],
+                      "neobolt==1.7.17", "Shapely==1.8.1", "six==1.16.0"],
```

Comment:
Is it necessary to pin the version at all? This could cause conflicts.
```suggestion
                      "neobolt==1.7.17", "Shapely>=1.7.1,<1.9", "six==1.16.0"],
```

Reply:
> Is it necessary to pin the version at all? This could cause conflicts.

I pray this will be accepted soon so i can use neomodel with python 3.10